ext/json/json.c

```diff
@@ -95,6 +96,9 @@ static PHP_MINIT_FUNCTION(json)
 	INIT_CLASS_ENTRY(ce, "JsonSerializable", json_serializable_interface);
 	php_json_serializable_ce = zend_register_internal_interface(&ce);
 
+	INIT_CLASS_ENTRY(ce, "JsonException", NULL);
```

Comment:
RFC just to resolve some inconsistency that's irrelevant due to case-insensitivity? I'm sorry I brought this up.